packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -187,7 +186,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
           location: reminder.booking.location || "",
           additionalNotes: reminder.booking.description,
           responses: responses,
-          meetingUrl: bookingMetadataSchema.parse(reminder.booking.metadata || {})?.videoCallUrl,
```

Comment:
How does it fix the error specifically?
Also, can we not simply make it`bookingMetadataSchema.parse(reminder.booking.metadata || {})?.videoCallUrl || ""`

Reply:
<img width="689" alt="Screenshot 2023-04-21 at 3 09 29 PM" src="https://user-images.githubusercontent.com/1780212/233603200-174880fb-85b6-4daf-b5cd-e40280e5173a.png">
Also, this might make the default as empty string everywhere.

Reply:
> <img alt="Screenshot 2023-04-21 at 3 09 29 PM" width="689" src="https://user-images.githubusercontent.com/1780212/233603200-174880fb-85b6-4daf-b5cd-e40280e5173a.png">
> 
> Also, this might make the default as empty string everywhere.

I tried that, but it's still throwing zod error "Expected string, received null"